src/libraries/System.Net.Http/src/System/Net/Http/HttpClient.cs

```diff
@@ -5,6 +5,7 @@
 using System.Diagnostics;
 using System.IO;
 using System.Net.Http.Headers;
+using System.Runtime.ExceptionServices;
```

Comment:
You're no longer using ExceptionDispatchInfo.  Do you still need this?

Reply:
Removed